extensions/wikia/CommunityPage/styles/components/_help.scss

```diff
@@ -6,23 +6,26 @@
 $small-icon-size: 18px;
 $big-icon-size: 30px;
 $help-module-margin: 12px;
-$help-module-padding-bottom: 20px;
+$help-module-padding: 24px;
+$help-module-padding-side: 18px;
 $help-module-link-padding: 12px;
+$help-module-title-padding: 3px;
 $policy-link-padding-top: 18px;
 $policy-link-margin-top: 14px;
```

Comment:
just a nitpick - when naming of consts, I'd use `spacing` rather than `padding` and `margin` cause afterwards we don't need to think whether is the one or another when we want to reuse it